vyper/codegen/expr.py

```diff
@@ -189,12 +190,18 @@ def __init__(self, node, context):
         self.ir_node.source_pos = getpos(self.expr)
 
     def parse_Int(self):
-        # Literal (mostly likely) becomes int256
+        propagated_type = self.expr._metadata.get("type")
```

Comment:
is it still possible for there not to be a type here?

Reply:
Yes, in a `for` loop for example:
```
@external
def foo():
    for i in range(5):
        pass
```

Reply:
are there are other situations? if this is the only one, we should probably fix it